classes/components/forms/FieldAuthors.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * @file classes/components/form/FieldAuthors.php
+ *
+ * Copyright (c) 2025 Simon Fraser University
+ * Copyright (c) 2025 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class FieldAuthors
+ *
+ * @ingroup classes_controllers_form
+ *
+ * @brief A field for authors.
+ */
+
+namespace PKP\components\forms;
+
+class FieldAuthors extends Field
+{
+    /** @copydoc Field::$component */
+    public $component = 'field-authors';
+
+    /** @copydoc Field::$component */
+    public $default = [];
+
+    /**
+     * Primary language
+     * Filled in components/ListPanel/ContributorsListPanel.vue
+     */
+    public string $primaryLocale = '';
```

Comment:
Just to clean up things bit more. I don't think thats necessary. Thats something thats being passed to the field on the vue side. But its not something you would want to configure when creating the field configuration in php. Same I think is true for supportedFormLocales below